lighter/__init__.py

```
@@ -9,3 +9,10 @@
 from .utils.logging import _setup_logging
 
 _setup_logging()
+
+# Expose Trainer from PyTorch Lightning for convenience
+from pytorch_lightning import Trainer
+
+from .engine.config import Config
```

Ruff (0.8.2)

`.engine.config.Config` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)